sound/soc/codecs/rt700.c

```diff
@@ -1240,10 +1241,15 @@ int rt700_io_init(struct device *dev, struct sdw_slave *slave)
 	rt700->hw_init = true;
 
 	/* Enable Runtime PM */
-	pm_runtime_set_autosuspend_delay(&slave->dev, 3000);
-	pm_runtime_use_autosuspend(&slave->dev);
-	pm_runtime_mark_last_busy(&slave->dev);
-	pm_runtime_enable(&slave->dev);
+	if (!rt700->first_init) {
+		pm_runtime_set_autosuspend_delay(&slave->dev, 3000);
+		pm_runtime_use_autosuspend(&slave->dev);
+		pm_runtime_mark_last_busy(&slave->dev);
+		pm_runtime_enable(&slave->dev);
+		rt700->first_init = true;
```

Comment:
one question: can you use "bool pm_runtime_enabled(struct device *dev)" instead of first_init ?

Reply:
@ranj063 can you help here? It's my understanding that the user can change the values and enable/disable pm_runtime via debugfs, so using the pm_runtime status to check if we need to enable pm_runtime seems a bit hairy.